presentations/interim.html

```diff
@@ -0,0 +1,77 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="css/reveal.css">
+    <link rel="stylesheet" href="css/simple.css" id="theme">
+    <link rel="stylesheet" href="css/github.css">
+</head>
+
+<body>
+    <div class="reveal">
+        <div class="slides">
+            <section>
+                <img src="../logo.svg" style="border: none;box-shadow: none;height: 10vh;" />
+                <h2>XMPP-Grid Broker</h2>
+                <h3>Interim Presentation</h3>
+                <small>Raphael Zimmermann &amp; Fabian Hauser</small>
+            </section>
+            <section>
+                <h3>🏋 The Problem</h3>
+                <img src="resources/explanatory-graphics-problem.svg" alt="problem explanatory graphic"
+                     style="height: 50vh;">
+            </section>
+            <section>
+                <h3>📚 IETF Draft</h3>
+                <p>"Using XMPP for Security Information Exchange"</p>
+                <img src="resources/explanatory-graphics-ietf-draft.svg" alt="ietf draft explanatory graphic"
+                style="height: 50vh;">
+            </section>
+            <section>
+                <h3>🎉 XMPP Grid Broker</h3>
+                <img src="resources/explanatory-graphics-xmpp-grid-broker.svg" alt="xmpp-grid-broker explanatory graphic"
+                style="height: 50vh;">
+            </section>
+            <section>
+                <h3>🗃 Task Description</h3>
+                <ul style="list-style-type: none;">
+                    <li>✓ Understand the Drafts and Standards (IETF, XEPs)</li>
+                    <li>✓ Collect Requirements</li>
+                    <li>✓ Architecture</li>
+                    <li>✓ Technology Selection</li>
+                    <li>👉 Implementation</li>
+                </ul>
+                <p>
+                    <small><a href="https://xmpp-grid-broker.github.io/documents/final-submission-document.pdf">Full Task Description</a></small>
+                </p>
+            </section>
+            <section>
```

Comment:
I think it would be nice to have a Infrastructure overview diagram here (or maybe even after the demo?), to show how the stack looks like in practice.

Reply:
What do you mean by Infrastructure overview diagram? Sth that shows docker or a C4 Diagram?

Reply:
A C4 diagram (or deployment diagram) that shows how everything is connected when the application is running (openfire -TLS- nginx -TLS- angular).

Reply:
We can use this one: https://github.com/xmpp-grid-broker/documents/blob/7e242f67b17527f919fa2c132a6feed9c79f69ff/documents/final-submission-document/resources/architecture_container_webapplication_with_http_proxy.pdf , what do you think?

Reply:
@raphiz 🎉 👍

Reply:
@fabianhauser done